### PR TITLE
Feat/st 399

### PR DESCRIPTION
**Pull request by muradalamv1**
Please remove this line and everything above and fill the following sections:

 https://tools.hmcts.net/jira/browse/ST-399

### Change description ###
Update code for Upload documents feast/ST-332
Created page for further details feastST-399
Made email as mandatory field for subject when cliked Email as preference type  feast/ST-326

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by osheaf01 — DISMISSED** [review]
LGTM

**Review by jaberelfasi — APPROVED** [review]

**Review by ElifDenizEsea — DISMISSED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #!groovy
-
+import uk.gov.hmcts.contino.AppPipelineConfig
```
Comment (johnoneill46) [inline Jenkinsfile_CNP:2]: Why the move?

File: config/pmd/ruleset.xml [hunk 2]
```diff
@@ -105,4 +105,10 @@
       <property name="cycloOptions" value="" />
     </properties>
   </rule>
+<!--  <rule ref="category/java/design.xml/ExcessiveMethodLength">-->
```
Comment (johnoneill46) [inline config/pmd/ruleset.xml:108]: Why remove this?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java [hunk 3]
```diff
@@ -328,6 +328,8 @@ public boolean isDivorce() {
         return divorceOrDissolution.isDivorce();
     }
 
+
```
Comment (johnoneill46) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java:331]: Remove please

File: config/pmd/ruleset.xml (hunk 2 above)
Comment (muradalamv1) [reply config/pmd/ruleset.xml]: It was added to resolve the exception but was not the part of the rules . i added it to test something and now removed it.

**Review by sjamihmcts — APPROVED** [review]

**Review by jaberelfasi — APPROVED** [review]

**Pull request merged**